apps/studio/prisma/schema.prisma

```diff
@@ -221,3 +222,33 @@ model Whitelist {
 
   @@index([email])
 }
+
+enum AuditLogEvent {
+  ResourceCreate
+  ResourceUpdate
+  ResourceDelete
+  ResourceMove
+  UserCreate
+  UserUpdate
+  UserDelete
+  Publish
+  Login
+  Logout
+  PermissionCreate
+  PermissionUpdate
+  PermissionDelete
+}
+
+model AuditLog {
+  id        Int           @id @default(autoincrement())
```

Comment:
issue: should we use BigInt for this? I presume that the `AuditLog` table would be the fastest growing table among all our tables in the DB.

Reply:
ohyea, yes, lemme update it!